recipes-bsp/u-boot/u-boot_2019.07.bbappend

```diff
@@ -39,6 +40,11 @@ do_configure_prepend_freedom-u540() {
     fi
 }
 
+do_compile_append_freedom-u540_sota() {
```

Comment:
this uses `sota` additional override, it that intentional ?

Reply:
Yes, this is aimed at OSTree which requires sota

Reply:
@alistair23 
OSTree is like a remote img server?

Reply:
No, it's an immutable rootFS. Kind of like git for your file system. https://ostree.readthedocs.io/en/latest/

Reply:
> Yes, this is aimed at OSTree which requires sota

ok then would it make sense to add the .in file to SRC_URI also under same override ?

Reply:
Good point!